libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2353,8 +2353,6 @@ string YulUtilFunctions::copyArrayFromStorageToMemoryFunction(ArrayType const& _
 
 string YulUtilFunctions::mappingIndexAccessFunction(MappingType const& _mappingType, Type const& _keyType)
 {
-	solAssert(_keyType.sizeOnStack() <= 1, "");
```

Comment:
Just curious if this had to do with us supporting only value type keys until some point in time?

Reply:
No, it was just laziness.